core/build.sbt

```diff
@@ -148,6 +149,12 @@ lazy val akkastreamTests =
           )
     )
     .settings(
+      (sourceGenerators in Test) += (avroScalaGenerateSpecific in Test).taskValue,
+      inConfig(Test)(sbtprotoc.ProtocPlugin.protobufConfigSettings),
+      PB.targets in Compile := Seq(
+        scalapb.gen() -> (sourceManaged in Compile).value / "sproto"
```

Comment:
whats the `sproto` there for?

Reply:
It's just a source managed directory name, where scala code is generated (scala proto)